framework/core/js/src/forum/components/PostStream.js

```diff
@@ -94,6 +95,15 @@ export default class PostStream extends Component {
       );
     }
 
+    // Allow extensions to add items to the end of the post stream.
+    if (viewingEnd) {
+      if (this.endItems().toArray().length) {
+        this.endItems()
+          .toArray()
+          .map((item) => items.push(item));
+      }
```

Comment:
I think we could simplify it to this

```suggestion
    if (viewingEnd) {
      items.push(...this.endItems().toArray());
```

Reply:
Thx! :)